patchouli_books/modded_for_dummies/en_us/entries/tweaks/tweaks_inv.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Inventory Tweaks",
+    "icon": "minecraft:chest",
+    "category": "patchouli:tweaks",
+    "pages": [
+        {
+            "type": "text",
+            "text": "$(#ffa500)$(l)Inventory Tweaks$(reset)’ main focus is, well, your inventory. Maybe it’s most famous addition is clicking $(k:invtweaks_sort_either.desc) to $(l)sort any inventory$(reset) you’re currently navigating."
+        },
+        {
+            "type": "text",
+            "text": "Other handy features introduced by the mod include auto-refilling items in your hotbar if you have more of them in your inventory and replacing a tool when it breaks, if you have another one ready."
```

Comment:
I think we lost this feature after E6 :<

Reply:
yep, seems like it